build.gradle.kts

```diff
@@ -0,0 +1,222 @@
+import groovy.util.Node
+import org.gradle.plugins.ide.idea.model.IdeaModel
+import org.robolectric.gradle.ShadowsPlugin.ShadowsPluginExtension
+
+// https://github.com/gradle/gradle/issues/21267
+val axtCoreVersion by project.extra { libs.versions.androidx.test.core.get() }
+val axtJunitVersion by project.extra { libs.versions.androidx.test.ext.junit.get() }
+val axtMonitorVersion by project.extra { libs.versions.androidx.test.monitor.get() }
+val axtRunnerVersion by project.extra { libs.versions.androidx.test.runner.get() }
+val axtTruthVersion by project.extra { libs.versions.androidx.test.ext.truth.get() }
+val espressoVersion by project.extra { libs.versions.androidx.test.espresso.get() }
+
+// For use of external initialization scripts...
+val allSdks by project.extra(AndroidSdk.ALL_SDKS)
+val configAnnotationProcessing by project.extra(emptyList<Project>())
+
+val thisVersion: String by project
+
+plugins {
+  alias(libs.plugins.android.library) apply false
+  alias(libs.plugins.detekt) apply false
+  alias(libs.plugins.error.prone)
+  alias(libs.plugins.idea)
+  alias(libs.plugins.kotlin.android) apply false
+  alias(libs.plugins.kotlin.jvm) apply false
+  alias(libs.plugins.robolectric.spotless)
+  alias(libs.plugins.robolectric.javadoc)
+  alias(libs.plugins.roborazzi) apply false
+}
+
+allprojects {
+  group = "org.robolectric"
+  version = thisVersion
+}
+
+project.afterEvaluate {
+  val ideaProject = rootProject.extensions.getByType<IdeaModel>().project
+  ideaProject.ipr.withXml {
```

Comment:
I didn't find any ipr file, so I don't know how to check if this is still working as expected.
If you have any tips for that, please let me know.

Reply:
Ok, IPR is the old format used by IntelliJ to store project settings: https://www.jetbrains.com/help/idea/creating-and-managing-projects.html#file-based